integration/stability/hypervisor_stability_kill_test.sh

```diff
@@ -21,18 +21,20 @@ PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
 # Set the runtime if not set already
 RUNTIME="${RUNTIME:-kata-runtime}"
 
+HYPERVISOR_NAME=$(basename ${HYPERVISOR_PATH})
```

Comment:
nit: you can use shell magic here like `${HYPERVISOR_PATH%/*}` or similar ;-)

Reply:
Wow, that is real magic ;)